src/components/o-policy-features/_preview.html

```diff
@@ -7,9 +7,14 @@
   </axa-policy-feature-item>
 
   <axa-policy-feature-item title="24/7 assistance"
-                           src="https://d5cplpsrt2s33.cloudfront.net/m/79774b901be10a5b/original/tick_white_web.svg"
                            alt="Assistance Svg Icon"
-                           description="We reward safe drivers : 75% no claims discount + an extra 10% off if you get a quote online">
+                           description="This svg is way too wide">
+    <svg class="js-m-policy-features-icon m-policy-feature-item__icon" xmlns="http://www.w3.org/2000/svg" width="1024" height="1024" viewBox="0 0 1024 1024">
```

Comment:
can we use axa-icon here? Just add the svg in the icons folder, give it a name that makes sense and per magic you can use it in the axa-icon. on aem, same stuff. how to use the icon there then, u can copy from cookie-disclaimer

Reply:
only if effort/benefit ratio makes sense

Reply:
these icons are rendered from DAM (per requirement), so axa-icon cannot be used.